shared/common-adapters/clickable-box.js.flow

```diff
@@ -14,6 +14,11 @@ export type Props = {|
   feedback?: boolean,
   // mobile only
   activeOpacity?: number,
+  // desktop only
```

Comment:
Flow complains about the shared/common-adapters/clickable-box.desktop.js changes without this